src/luamtev.c

```diff
@@ -282,5 +282,15 @@ int main(int argc, char **argv, char **envp) {
   mtev_main(APPNAME, config_file, debug, foreground,
             MTEV_LOCK_OP_LOCK, NULL, droptouser, droptogroup,
             child_main);
+
+  free(function);
```

Comment:
Bunch of leaks:

Example: https://scan7.coverity.com/reports.htm#v20002/p12027/fileInstanceId=30140689&defectInstanceId=6723260&mergedDefectId=1452114